web-api/dynamo-export.js

```diff
@@ -18,5 +18,8 @@ documentClient
   })
   .promise()
   .then(documents => {
+    documents.Items.sort((a, b) => {
```

Comment:
Did this end up helping make exporting local data easier to diff? 🥳 🎖️

Reply:
yeah it was easier to see where things were added / removed I think